modules/efk-logging-fluentd-undefined.adoc

```diff
@@ -5,22 +5,18 @@
 [id="efk-logging-fluentd-undefined_{context}"]
 = Configuring how the log collector normalizes logs 
 
```

Comment:
This section is primarily about how to deal with problems that arise when using `MERGE_JSON_LOG=true`.  It can be extremely useful to have your applications log their output in JSON, and have OpenShift logging automatically parse and index the data in Kibana, but this leads to several problems: field names may be empty, or contain characters that are illegal in Elasticsearch; different applications in the same namespace may output the same field name with different value data types; applications may emit too many fields; fields may conflict with the OpenShift logging built-in fields, and more.

Reply:
Outdated